Majasdarbs/03-majasdarbs/03-katrina-tiltina/index.html

```diff
@@ -0,0 +1,213 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/Majasdarbs/03-majasdarbs/03-katrina-tiltina/style.css">
```

Comment:
Wrong path to css. Should be 
<link rel="stylesheet" href="style.css">

https://www.w3schools.com/html/html_filepaths.asp